modules/appeals_api/catalog-info.yaml

```diff
@@ -1,11 +1,21 @@
 apiVersion: backstage.io/v1alpha1
-kind: Location
+kind: API
 metadata:
-  name: appeals-apis
-  description: A collection Appeal APIs
+  name: decision-reviews
+  description: The Decision Reviews API allows you to interact with a Veteran’s decision reviews, also known as benefit appeals.
+  title: Decision Reviews API
   tags:
-    - department-of-veterans-affairs
+    - internal
+    - lighthouse
+    - ruby
+  links:
+    - url: https://developer.va.gov/explore/api/decision-reviews
+      title: Public Docs
+      icon: web
 spec:
-  targets:
-    - ./config/backstage/appeals-status.yaml
-    - ./config/backstage/decision-reviews.yaml
+  type: openapi
+  lifecycle: production
+  owner: lighthouse-banana-peels
+  system: vets-api
+  definition:
+    $json: https://api.va.gov/services/appeals/docs/v2/decision_reviews
```

Comment:
[Based on the backstage docs](https://backstage.io/docs/features/software-catalog/descriptor-format/#substitutions-in-the-descriptor-format), I think defining `$json` here will allow the site to fetch our JSON docs an render them correctly. I can see that the [Benefits Intake config](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/vba_documents/catalog-info.yaml#L23), for example, `$text` is being used, which seems to work fine, but I saw `$json` in the backstage docs and it seemed most appropriate.